src/data/resample_uscabdry.py

```diff
@@ -0,0 +1,162 @@
+# Copyright 2018 SAS Project Authors. All Rights Reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+
+"""Resample US/Canadian border.
```

Comment:
Do we need to do the same for US-MEX border as well ?

Reply:
To my knowledge there is no border test for US-Mexico border. 
So we dont need to do any logic for that in the test harness

@kate-harrison , for commenting on this further

Reply:
Confirmed, we do not need to do this for the Mexican border.